### PR TITLE
Integrate drone.astronomer.io

### PR DESCRIPTION
**Pull request by andriisoldatenko**
- During release process we need to build operator image to docker hub, we can do it in this repo or may be in astronomer/astronomer during docker push platform: 🤔 

TODO:
- [x] Integrate with drone.io
https://drone.astronomer.io/astronomer/airflow-operator
- [x] Add operator-sdk Dockerfile to astronomer/astronomer/dockerfiles
https://github.com/astronomer/astronomer/pull/262
- [x] Create operator-sdk helm chart in astronomer/helm.astronomer.io repo.
https://github.com/astronomer/helm.astronomer.io/pull/116/files
- [ ] Add a drone build step to `astronomer/astronomer/drone.yml` to trigger the `helm.astronomer.io` build

### CODE REVIEW

File: hack/push-image [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```
Comment (schnie) [inline hack/push-image]: @andriisoldatenko whats the reasoning for the `hack` directory here? Is it really hacky or could we just call it `bin` or something?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
+
+# Public repository for images
+REPOSITORY ?= astronomerinc
+
+# Bump this on subsequent build, reset on new version or public release. Inherit from env for CI builds
+BUILD_NUMBER ?= 1
+
+# HELM_REPOSITORY := helm.astronomer.io
```
Comment (schnie) [inline Makefile]: Can probably get rid of this commented line.

File: hack/push-image (hunk 1 above)
Comment (andriisoldatenko) [reply hack/push-image]: I can rename it to `bin`, in k8s and golang itself we are using `bin` for build artifacts and `hack` for bash scripts -> https://github.com/kubernetes/kubernetes/tree/master/hack

File: Makefile (hunk 2 above)
Comment (andriisoldatenko) [reply Makefile]: yes on it.

Comment (andriisoldatenko) [reply Makefile]: fixed.

File: hack/push-image (hunk 1 above)
Comment (andriisoldatenko) [reply hack/push-image]: fixed.

**Comment by schnie** [comment]
Looks good!

**Pull request merged**